src/reference-data/types.ts

```diff
@@ -19,6 +19,7 @@ export type PreassignedFareProduct = {
     userProfileRefs: string[];
     appVersionMin: string | undefined;
     appVersionMax: string | undefined;
+    latestActivationDate?: string;
```

Comment:
I think this should be `number`? Don't understand why it is a `string` 🤔

Reply:
I guess number would be more correct, and we'll save a conversion later. 

Good catch, typically defaulting to strings as they are not as easily changed in case they start with zeroes etc., but shouldn't be an issue here.